v3/mechlovin/infinity875/infinity875.json

```diff
@@ -1,45 +1,152 @@
 {
-  "name": "IF875",
+  "name": "IF875 Rev. 1",
   "vendorId": "0x4D4C",
   "productId": "0x0875",
-  "keycodes": ["qmk_lighting"],
```

Comment:
You will probably want to keep this line of code so that the LIGHTING tab shows up

![image](https://user-images.githubusercontent.com/17491233/233784939-4668b180-bf4f-4f87-a42d-1e34471b3b54.png)